templates/centos-stream9.tpl.yaml

```diff
@@ -49,13 +49,14 @@ metadata:
 {% if item.default %}
     template.kubevirt.io/default-os-variant: "true"
 {% endif %}
+    architecture: "{{ 'amd64' if target_arch == 'x86_64' else target_arch }}"
```

Comment:
The label should be prefixed with `template.kubevirt.io/`

Reply:
Done.